src/main/kotlin/carracing/Race.kt

```diff
@@ -0,0 +1,20 @@
+package carracing
+
+class Race(
+    private val numCars: Int,
+    private val numRounds: Int,
+) {
+    private val cars = List(numCars) { Car() }
+
+    fun start() {
+        for (round in 1..numRounds) {
+            cars.forEach { it.move() }
+            displayRace()
+            println()
+        }
+    }
```

Comment:
I feel like this method is doing too much. How about dividing its responsibilities?
> If you split your code by responsibility, how do you approach writing tests for it?